rmgpy/chemkin.pyx

```diff
@@ -73,10 +73,10 @@ def readThermoEntry(entry, Tmin=0, Tint=0, Tmax=0):
     the label of the species and the thermodynamics model as a :class:`NASA`
     object.
     
-    Format specification at http://www2.galcit.caltech.edu/EDL/public/formats/chemkin.html
+    Format specification at http://akrmys.com/public/chemkin/CKm_inp.html.en#THRM
```

Comment:
I don't think this is a good link. It might be better to just cite the Chemkin Input Manual, even though it may not be accessible to people who don't have Chemkin.

Reply:
fixed